content/4_Gold/PURS.mdx

```diff
@@ -922,13 +1314,11 @@ before using a BIT or segment tree (though
 
 </Info>
 
-### General
+## General
 
 <Problems problems="practice" />
 
-### USACO
-
-<!-- Haircut, Balanced Photo, and Circle Cross are just variations on inversion counting. -->
+## USACO
```

Comment:
lol we should move this hint to the problem edi

Reply:
no yeah i'll do that in a separate pr